examples/Makefile

```diff
@@ -416,6 +416,31 @@ ifeq ($(PLATFORM),PLATFORM_DESKTOP_SDL)
         LDLIBS += -latomic
     endif
 endif
+ifeq ($(PLATFORM),PLATFORM_DESKTOP_RGFW)
+    ifeq ($(PLATFORM_OS),WINDOWS)
+        # Libraries for Windows desktop compilation
+        LDLIBS = -lopengl32 ..\src\libraylib.a -lgdi32 -lgdi32 -lXinput -lopengl32 -lShcore  
```

Comment:
Why are `-lXinput` and `-lShcore` required? Also, `-lopengl32` is linked twice while it does not seem to be required with GLFW.

Reply:
Xinput is used for controller input and I believe Shcore is used for monitor functions.

The main reason why GLFW doesn't need to link with -lopengl32 twice is because it loads WGL directly into itself. I do not like that design.